### PR TITLE
Core 1870

### PR DESCRIPTION
**Pull request by sboleyn**
Implement code review from PR #509. Add ErroredListing and SubscriptionEmptyListing stories.

### CODE REVIEW

**Review by slr71 — APPROVED** [review]
Looks good! 👍 I didn't catch any issues, but I had a couple of questions about some field labels that I was expecting to be internationalized.

File: stories/subscriptions/Listing.stories.js [hunk 1]
```diff
@@ -86,18 +94,35 @@ function sortSubscriptionsByUsername(subscriptions, sortAscending) {
...
+    mockAxios.onGet(subscriptionListingUriRegexp).reply(200, emptyListing);
+    return <ListingTest isAdminView={isAdminView} />;
+};
+
+/**
+ * Covers the case where the subscription listing endpoint returns an error.
+ */
+export const ErroredListingTest = ({ isAdminView }) => {
+    mockAxios.onGet(subscriptionListingUriRegexp).reply(400, erroredListing);
+    return <ListingTest isAdminView={isAdminView} />;
```
Comment (psarando) [inline stories/subscriptions/Listing.stories.js:119]: 👍

File: src/components/subscriptions/listing/TableView.js [hunk 2]
```diff
@@ -136,119 +267,20 @@ function TableView(props) {
...
+                                onDetailsSelected={onDetailsSelected}
+                                onEditQuotasSelected={onEditQuotasSelected}
+                                onEditSubscriptionSelected={
+                                    onEditSubscriptionSelected
+                                }
+                                subscriptions={subscriptions}
+                                selected={selected}
+                                t={t}
+                                tableId={tableId}
+                            />
```
Comment (psarando) [inline src/components/subscriptions/listing/TableView.js:283]: ✨

File: src/components/subscriptions/edit/EditQuotas.js [hunk 3]
```diff
@@ -143,28 +135,22 @@ function EditQuotasDialog(props) {
...
-                required
-                validate={(value) => nonEmptyField(value, i18nUtil)}
-            />
+            <SimpleExpansionPanel
+                parentId={parentId}
+                header={t("details")}
+                defaultExpanded={true}
+            >
+                <Grid container spacing={2}>
+                    <GridLabelValue label="Username">
```
[the remark this answers is not in the document]
Comment (psarando) [reply src/components/subscriptions/edit/EditQuotas.js:146]: I think this should use the i18n translations:
```suggestion
                    <GridLabelValue label={t("username")}>
```

File: src/components/subscriptions/edit/EditQuotas.js [hunk 4]
```diff
@@ -143,28 +135,22 @@ function EditQuotasDialog(props) {
...
+                        <Typography>{subscription?.user?.username}</Typography>
+                    </GridLabelValue>
+                    <GridLabelValue label="Plan Name">
```
[the remark this answers is not in the document]
Comment (psarando) [reply src/components/subscriptions/edit/EditQuotas.js:149]: I think this should also use the i18n translations:
```suggestion
                    <GridLabelValue label={t("planName")}>
```

**Review by psarando — CHANGES_REQUESTED** [review]
Very nice cleanup and refactoring here. ✨ 

LGTM, except I think those labels should use i18n messages.

File: src/components/subscriptions/edit/EditQuotas.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (sboleyn) [reply src/components/subscriptions/edit/EditQuotas.js]: Thank you for catching these!

**Review by psarando — APPROVED** [review]
LGTM :shipit: 👍

**Pull request merged**